Scripts/Services/VisualRecognition/V4/Model/BaseCollectionTrainingStatus.cs

```diff
@@ -0,0 +1,33 @@
+/**
+* Copyright 2018, 2019 IBM Corp. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*/
+
+using Newtonsoft.Json;
+
+namespace IBM.Watson.VisualRecognition.V4.Model
+{
+    /// <summary>
+    /// Training status information for the collection.
+    /// </summary>
+    public class BaseCollectionTrainingStatus
+    {
```

Comment:
I don't think this model should be here - did you delete your models before generating?